server/base/src/main/java/org/apache/accumulo/server/master/recovery/HadoopLogCloser.java

```diff
@@ -36,7 +36,7 @@
 
   public HadoopLogCloser() {
     log.warn("{} has been deprecated. Please update property {} to {} instead.",
-        getClass().getName(), Property.MANAGER_WALOG_CLOSER_IMPLEMETATION.getKey(),
+        getClass().getName(), Property.MANAGER_WAL_CLOSER_IMPLEMENTATION.getKey(),
```

Comment:
I don't think you need to worry about resolving here. What you have is fine since this whole class was just deprecated in main. You just replaced the new property with a newer one.

Reply:
This is one of those props that got changed multiple times in the main branch. This warning was originally added to help facilitate the migration from `master.` to `manager.`. We can keep the`manager.walog.` version around to help facilitate that transition, but the log message should really only refer to the new `manager.wal.` name. So, I think this change is good here.